### PR TITLE
task 17

### PR DESCRIPTION
**Pull request by yana-may25**

### CODE REVIEW

File: src/task17/task17 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    lock() { this.locked = true; },
+    _content: [1, 2, 3],
+    get content() {
+        if (this.locked) throw new Error("Locked!");
+        return this._content;
+    }
+};
+
+const withBoxUnlocked = body => {
+    let isLocked = box.locked;
```
Comment (MrOlegKravchenko) [inline src/task17/task17]: isLocked does not make code easy, my opinion

File: src/task17/task17 [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const box = {
+    locked: true,
+    unlock() { this.locked = false; },
+    lock() { this.locked = true; },
+    _content: [1, 2, 3],
```
Comment (MrOlegKravchenko) [inline src/task17/task17]: remove 1, 2, 3

**Comment by MrOlegKravchenko** [comment]
Correct and merge

File: src/task17/task17 (hunk 1 above)
Comment (yana-may25) [reply src/task17/task17]: you want it to be wasLocked, don't you?

File: src/task17/task17 (hunk 2 above)
Comment (yana-may25) [reply src/task17/task17]: yep, it was in the e-book as well:)

**Pull request merged**